saleor/asgi/__init__.py

```diff
@@ -32,7 +33,8 @@ def preload_app() -> None:
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "saleor.settings")
 
 application = get_asgi_application()
-application = health_check(application, "/health/")  # type: ignore[arg-type] # Django's ASGI app is less strict than the spec # noqa: E501
+application = otel_context(application)  # type: ignore[arg-type] # Django's ASGI app is less strict than the spec # noqa: E501
```

Comment:
Shouldn't this be the outermost middleware? What if other middleware uses telemetry?